requirements.txt

```diff
@@ -1,4 +1,4 @@
-carto>=1.3.1
+carto>=1.5.0
```

Comment:
While in development let's install from master like in https://github.com/CartoDB/cartoframes/pull/576

Reply:
Thanks!